src/plugins/intel_cpu/src/transformations/cpu_opset/x64/pass/mlp_fusion.cpp

```diff
@@ -45,7 +45,7 @@ ov::intel_cpu::MLPFusion::MLPFusion() {
 
     auto result = down_proj;
 
-    matcher_pass_callback callback = [OV_CAPTURE_CPY_AND_THIS](ov::pass::pattern::Matcher& m) {
+    matcher_pass_callback callback = [&](ov::pass::pattern::Matcher& m) {
```

Comment:
need to use [=] instead, otherwise pattern node shared-ptr will be empty when call-back happens.

Reply:
Updated.